src/OpenTelemetry.Exporter.Geneva/GenevaLogExporter.cs

```diff
@@ -329,7 +375,7 @@ internal int SerializeLogRecord(LogRecord logRecord)
         cntFields += 1;
 
         cursor = MessagePackSerializer.SerializeAsciiString(buffer, cursor, "name");
-        cursor = MessagePackSerializer.SerializeUnicodeString(buffer, cursor, name);
+        cursor = MessagePackSerializer.SerializeUnicodeString(buffer, cursor, categoryName);
```

Comment:
Not related to this PR: Can `LogRecord.CategoryName` have non-ASCII characters?

Reply:
I think they could potentially include non-ASCII characters yes. There is `LoggerFactory.CreateLogger(string categoryName)` API where you are free to specify a category name using whatever characters, but you can also have unicode in class names which can become category names through `ILogger<T>`. Ex:

```csharp
        public class MyClassû
        {
            private readonly ILogger<MyClassû> logger;
        }
```

Reply:
Thanks @CodeBlanch !

I think in that case we should also update the code here: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/src/OpenTelemetry.Exporter.Geneva/GenevaLogExporter.cs#L312 to expect Unicode characters for `LogRecord.Exception.GetType().FullName` as the user might create a custom exception class with Unicod characters in its name.